### PR TITLE
payloads/external/edk2: Add MTRR programming option

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

File: payloads/external/edk2/Makefile [hunk 1]
```diff
@@ -121,10 +121,14 @@ endif
 ifeq ($(CONFIG_EDK2_TPM_ENABLE),y)
 BUILD_STR += -D TPM_ENABLE=TRUE
 endif
+# NETWORK_LAN_ROM		= FALSE
```
Comment (macpijan) [inline payloads/external/edk2/Makefile:124]: Commit says only about disabling MTRR. Do we also need that line here?

Comment (miczyg1) [reply payloads/external/edk2/Makefile:124]: Forgot to add the comment with the default value in previous commits/PRs for this option. Good to have but not needed.

Comment (macpijan) [reply payloads/external/edk2/Makefile:124]: Maybe it should be a separate commit then, if note related to the MTTR option? Would make it easier when rebasing and cleaning up patches.

Comment (miczyg1) [reply payloads/external/edk2/Makefile:124]: Made it a separate commit

Comment (macpijan) [reply payloads/external/edk2/Makefile:124]: Thanks

**Pull request merged**